docs/source/reference/pysupported.rst

```diff
@@ -190,11 +190,32 @@ use-case would look like:
      handle_error_case()
      return error_code
 
-The ``finally`` block and the ``else`` block of a ``try .. except`` are
-supported.
+``try .. except .. else .. finally``
+''''''''''''''''''''''''''''''''''''
+
+The ``else`` block and the ``finally`` block of a ``try .. except`` are
+supported:
 
-The ``try .. finally`` construct without the ``except`` is currently
-**unsupported**.
+  .. code-block:: python
+
+    >>> @jit(nopython=True)
+    ... def foo():
+    ...     try:
+    ...         print('main block')
+    ...     except Exception:
+    ...         print('handler block')
+    ...     else:
+    ...         print('else block')
+    ...     finally:
+    ...         print('final block')
+    ...
+    >>> foo()
+    main block
+    handler block
```

Comment:
Should this not be `else block`.  I don't see  any exception being raised that would trigger execution of the `handler block`?

Reply:
you're right